docs/installation/sources/sources_linux.rst

```diff
@@ -69,6 +69,10 @@ Dependencies
 * :ref:`gtest_sl` [optional]
 * :ref:`pythonreq_sl` [optional]
 
+.. seealso::
+  For further information about this Fast DDS version dependencies and other products related versions, please check
+  the Fast DDS :ref:`related_versions` section.
```

Comment:
If the suggestion  about the name of the `related_versions` section is applied, please change also here. Moreover, if we change the name we can refer directly to the specific section inside the page, i.e. `Build time dependencies`